### PR TITLE
:sparkle: add option for additional info banner in dialog

### PR DESCRIPTION
**Pull request by Brynjulf**

### CODE REVIEW

File: src/molecules/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
       children,
       width,
       actions,
-      withContentPadding = true,
+      withContentPaddingX = true,
+      withContentPaddingY = true,
       withBorders = false,
       contentMaxHeight = '60vh',
+      hideClose,
```
Comment (mariush2) [inline src/molecules/Dialog/Dialog.tsx:80]: Couldn't hideClose instead be achieved with onClose being optional?

File: src/molecules/Dialog/Dialog.tsx [hunk 2]
```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
       children,
       width,
       actions,
-      withContentPadding = true,
+      withContentPaddingX = true,
+      withContentPaddingY = true,
```
Comment (mariush2) [inline src/molecules/Dialog/Dialog.tsx:77]: Prefer to have these as 1 prop like this:
```
  withContentPadding: {
     height: true,
     width: true,
  }
```

File: src/molecules/Dialog/Dialog.tsx [hunk 3]
```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
...
+            publication to our partners.
+          </Typography>
+          <Typography>
+            When the selected group of experiences are ready to be publish,
+            check the last checkbox that says &#39;Selected set of experiences
+            are ready for publishing&#39;. This will notify the publisher that
+            you are ready to publish them.
+          </Typography>
+        </div>
+      ),
```
Comment (mariush2) [inline src/molecules/Dialog/Dialog.tsx:94]: This shouldn't be the default value of additionalInfo

**Review by mariush2 — CHANGES_REQUESTED** [review]

File: src/molecules/Dialog/Dialog.tsx (hunk 3 above)
Comment (Brynjulf) [reply src/molecules/Dialog/Dialog.tsx]: Quite right 😅 removed

File: src/molecules/Dialog/Dialog.tsx (hunk 2 above)
Comment (Brynjulf) [reply src/molecules/Dialog/Dialog.tsx]: Changed to reflect that pattern

File: src/molecules/Dialog/Dialog.tsx (hunk 1 above)
Comment (Brynjulf) [reply src/molecules/Dialog/Dialog.tsx:80]: `onClose` is also used by the cancel button, but i added a comment in the property description to specify this, hope that works?

**Review by mariush2 — APPROVED** [review]

**Pull request merged**